test/Api/CatalogApiTest.php

```diff
@@ -0,0 +1,640 @@
+<?php
+namespace SquareConnect\Api;
+
+use \SquareConnect\Configuration;
+use \SquareConnect\ApiClient;
+use \SquareConnect\ApiException;
+use \SquareConnect\ObjectSerializer;
+use \SquareConnect\Model\CatalogObject;
+use \SquareConnect\Model\BatchUpsertCatalogObjectsRequest;
+use \SquareConnect\Model\BatchDeleteCatalogObjectsRequest;
+use \SquareConnect\Model\UpdateItemTaxesRequest;
+use \SquareConnect\Model\UpsertCatalogObjectRequest;
+
+
+error_reporting(E_ALL & ~E_NOTICE);
```

Comment:
Are you going to keep this here? Removing errors by doing this, means it could happen to a developer while they use it in prod